test/integration/package.json

```diff
@@ -1,6 +1,6 @@
 {
     "testDependencies": {
-        "selenium-webdriver": "2.44.0",
-        "chromedriver": "2.27.0"
+        "selenium-webdriver": "2.45.0",
+        "chromedriver": "2.33.0"
```

Comment:
I just noticed these are in `testDependencies` which doesn't seem to be a real thing? Or is it? I would expect `devDependencies`.

Reply:
We didn't put them in devDependencies because we didn't want to force everyone to download them. But we did want to document them somewhere, so we put them in testDependencies.

Reply:
I think we're safe from people needing to install them by virtue of this being in a separate `package.json`. I just updated it to `devDependencies` and it didn't install them in the top-level `npm install`.